src/App.js

```diff
@@ -29,30 +29,41 @@ class App extends Component {
       this.getActivePlayers(allPlayers)
     }
   }
-  
+
+  showFrontOrBack = (sideToShow) => {
+      this.setState({ showCardSide: sideToShow})
+  }
+
+  setCurrentPlayer = (chosenPlayer) => {
+    this.setState({ chosenPlayer: chosenPlayer })
```

Comment:
Get the chosen player from <Find>, set it in state in <App> so that it can be passed around to all the other components. I like the way Redux handles this better, it makes more sense to me.